root/client/src/components/monthlyTable/calendar.js

```diff
@@ -0,0 +1,52 @@
+import { React } from 'react';
+import { Week } from './week.js';
+
+export const Calendar = ({ pollAvailabilities, year, month, firstDayOfMonth, daysInMonth, handleDayClick }) => {
+  const arrCalendar = [["", "", "", "", "", "", ""], ["", "", "", "", "", "", ""], ["", "", "", "", "", "", ""], ["", "", "", "", "", "", ""], ["", "", "", "", "", "", ""], ["", "", "", "", "", "", ""]];
```

Comment:
what is this exactly?

also, i think you can programatically create this better using an array function

Reply:
Removed the array literal, it should look better.